dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -108,20 +115,42 @@ private static String osDetectedArch() {
     }
   }
 
-  private static DependencyNode resolveCompileTimeDependencies(
-      DependencyNode rootDependencyArtifact)
+  /**
+   * @param mavenRepositoryUrls Maven repository URLs to search for dependencies
+   * @param addMavenCentral if true, add Maven Central to the end of the repository list
+   * @throws IllegalArgumentException if a URL is malformed or not having an allowed scheme
+   */
+  public DependencyGraphBuilder(Iterable<String> mavenRepositoryUrls, boolean addMavenCentral) {
```

Comment:
DependencyGraphBuilder to take configuration of repository URLs. As of now, a static field of `RepositoryUtility` holds the repository URLs. I believe they should be an instance variable of DependencyGraphBuilder.

From old comment on the static mutable field in by Elliotte:

> Mutable static state suggests we're growing past the point where this class can plausibly be static. OK, for now but consider whether refactoring might help. Either to use more instance methods or to pass the repsositoryURLs as arguments to a method instead.
https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/349/files#r249024802